library/src/main/java/com/datatorrent/lib/dimensions/aggregator/Aggregator.java

```diff
@@ -0,0 +1,14 @@
+package com.datatorrent.lib.dimensions.aggregator;
+
+import com.datatorrent.lib.dimensions.AggregateEvent;
+
+import gnu.trove.strategy.HashingStrategy;
+
+public interface Aggregator<EVENT, AGGREGATE extends AggregateEvent> extends HashingStrategy<EVENT>
```

Comment:
+1 This is a rare case where I agree with nesting the interface. An Aggregator will never be used without an AggregateEvent.